src/conf.c

```diff
@@ -75,8 +88,8 @@ int conf_loadfile( conf_t *conf, char *file )
 		line ++;
 
 		*s = 0;
-		fscanf( fp, "%100[^\n#]s", s );
-		fscanf( fp, "%*[^\n]s" );
+		axel_fscanf( fp, "%100[^\n#]s", s );
+		axel_fscanf( fp, "%*[^\n]s" );
```

Comment:
This call should be merged with the previous one, as well as the call to `fgetc` following it.

Reply:
As I wrote in the other pull request:

> I don't think you can merge those calls: if the first pattern is not matched (i.e. line starting with a comment), no matching for the second pattern will be attempted, thus preventing the file pointer from proceeding forward. I believe this is the reason why those calls are split on 3 different lines.

Reply:
The pattern on the second line can match zero characters too, so it's no problem.

Reply:
Also, spaces could be trimmed in this stage by adding the corresponding directive here.

Reply:
axel_fscanf( fp, " %100[^\n#]s%*[^\n]s%*c", s );

that would skip spaces at the beginning too.

Reply:
well, perhaps adding space directives isn't a good idea in the sense fscanf wouldn't tell us how many newlines it found, so we can't update `line`.

Reply:
Sure, but imagine having the scanf format modified like you said: "%100[^\n#]s%*[^\n]s"
Now imagine the line to parse is "# some comments".
In this case the first format string won't match and the scanf will stop and exit there. The "rest of the line" won't be matched and fp won't move forward. At least this is what I got after doing various tests.

Reply:
`"%*[ \t]s"` might work as a replacement.

Reply:
initial white characters are discarded automatically. I don't think we need to care about that?

Reply:
hhmm, no, nevermind, a failure on the left side prevents the evaluation of following directives, so it wouldn't work

Reply:
correct, that's what I was trying to convey above